src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -49,8 +50,11 @@ public String greeting(
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "When thee application lists all available books.")
     })
-    public Iterable<Book> findAll() {
-        return bookRepository.findAll();
+    public Iterable<Book> findAll(BookFilters filters) {
```

Comment:
I think the usage of a BookFilters class is not necessary, and could be considered overdesign. You should receive all query params separately or a Map including all of them